osgp/shared/osgp-ws-smartmetering/src/main/resources/SmartMeteringInstallation.wsdl

```diff
@@ -145,48 +144,48 @@
       <wsdl:input message="tns:CoupleMbusDeviceRequest" name="CoupleMbusDeviceRequest">
       </wsdl:input>
       <wsdl:output message="tns:CoupleMbusDeviceAsyncResponse"
-        name="CoupleMbusDeviceAsyncResponse">
+                   name="CoupleMbusDeviceAsyncResponse">
       </wsdl:output>
     </wsdl:operation>
     <wsdl:operation name="GetCoupleMbusDeviceReponse">
       <wsdl:input message="tns:CoupleMbusDeviceAsyncRequest"
-        name="CoupleMbusDeviceAsyncRequest">
+                  name="CoupleMbusDeviceAsyncRequest">
       </wsdl:input>
       <wsdl:output message="tns:CoupleMbusDeviceResponse"
-        name="CoupleMbusDeviceResponse">
+                   name="CoupleMbusDeviceResponse">
       </wsdl:output>
     </wsdl:operation>
 
     <wsdl:operation name="DecoupleMbusDevice">
       <wsdl:input message="tns:DecoupleMbusDeviceRequest"
-        name="DecoupleMbusDeviceRequest">
+                  name="DecoupleMbusDeviceRequest">
       </wsdl:input>
       <wsdl:output message="tns:DecoupleMbusDeviceAsyncResponse"
-        name="DecoupleMbusDeviceAsyncResponse">
+                   name="DecoupleMbusDeviceAsyncResponse">
       </wsdl:output>
     </wsdl:operation>
     <wsdl:operation name="GetDecoupleMbusDeviceReponse">
       <wsdl:input message="tns:DecoupleMbusDeviceAsyncRequest"
-        name="DecoupleMbusDeviceAsyncRequest">
+                  name="DecoupleMbusDeviceAsyncRequest">
       </wsdl:input>
       <wsdl:output message="tns:DecoupleMbusDeviceResponse"
-        name="DecoupleMbusDeviceResponse">
+                   name="DecoupleMbusDeviceResponse">
       </wsdl:output>
     </wsdl:operation>
-    
-        <wsdl:operation name="CoupleMbusDeviceByChannel">
+
+    <wsdl:operation name="CoupleMbusDeviceByChannel">
       <wsdl:input message="tns:CoupleMbusDeviceByChannelRequest" name="CoupleMbusDeviceByChannelRequest">
       </wsdl:input>
       <wsdl:output message="tns:CoupleMbusDeviceByChannelAsyncResponse"
-        name="CoupleMbusDeviceByChannelAsyncResponse">
+                   name="CoupleMbusDeviceByChannelAsyncResponse">
       </wsdl:output>
     </wsdl:operation>
     <wsdl:operation name="GetCoupleMbusDeviceByChannelResponse">
       <wsdl:input message="tns:CoupleMbusDeviceByChannelAsyncRequest"
-        name="CoupleMbusDeviceByChannelAsyncRequest">
+                  name="CoupleMbusDeviceByChannelAsyncRequest">
       </wsdl:input>
       <wsdl:output message="tns:CoupleMbusDeviceByChannelResponse"
-        name="CoupleMbusDeviceByChannelResponse">
+                   name="CoupleMbusDeviceByChannelResponse">
```

Comment:
It looks like you are using some other formatting settings than have been used before, leading to lots of distracting formatting updates.

Reply:
yes, the formatter is giving us headaches, actually I formatted all files based on our last version (from last week) .....